samples/matter/light_switch/src/app_config.h

```diff
@@ -8,13 +8,19 @@
 
 /* ---- Lighting Example App Config ---- */
 
+#include "board_util.h"
+
 #define FUNCTION_BUTTON DK_BTN1
 #define FUNCTION_BUTTON_MASK DK_BTN1_MSK
+#if NUMBER_OF_BUTTONS == 2
```

Comment:
Where is this calculated?

Reply:
Here: https://github.com/nrfconnect/sdk-nrf/blob/main/samples/matter/common/src/board_util.h#L15